internal/resource/http.go

```diff
@@ -263,19 +263,24 @@ func (f *Fetcher) newHttpClient() error {
 	return nil
 }
 
-// getReaderWithHeader performs an HTTP GET on the provided URL with the
+// httpReaderWithHeader performs an HTTP Method on the provided URL with the
 // provided request header and returns the response body Reader, HTTP status
 // code, a cancel function for the result's context, and error (if any). By
 // default, User-Agent is added to the header but this can be overridden.
```

Comment:
```suggestion
// httpReaderWithHeader performs an HTTP request on the provided URL with the
// provided request header & method and returns the response body Reader, HTTP
// status code, a cancel function for the result's context, and error (if any). By
// default, User-Agent is added to the header but this can be overridden.
```
WDYT?